### PR TITLE
External preview feedback

### PR DESCRIPTION
**Pull request by vanessa-mplsfed**
Let's use this branch to capture any suggested adjustments during external previews.

### CODE REVIEW

**Comment by misha-mplsfed** [comment]
> Let's use this branch to capture any suggested adjustments during external previews.

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ On this resource, you will be able to access all of the survey support resources
 * [Questions](/README.md#questions)
  
 ## Getting Started
-Add our emails to your Safe Senders.  To ensure messages with important information don’t get lost, save the "@mpls.frb.org" domain to to your Safe Senders list. 
+Add our emails to your Safe Senders. To ensure messages with important information don’t get lost, save the "@mpls.frb.org" domain to to your Safe Senders list. 
 
 
 ## Events
```
Comment (htrostle-mplsfed) [inline README.md:39]: We will be adjusting the dates of these events.

File: README.md [hunk 2]
```diff
@@ -16,13 +16,13 @@ On this resource, you will be able to access all of the survey support resources
 
 | Resource | Purpose |
 |----------|---------|
-| [01 FAQ Overview](/01%20FAQ%20Overview.md) | Answers to common questions about the Survey of Native Nation and CICD |
+| [01 FAQ Overview](/01%20FAQ%20Overview.md) | Answers to common questions about the Survey of Native Nations and CICD |
 | [02 Okta How-to Guide](/02%20Okta%20How-to%20Guide.md) | Learn why we use Multifactor Authentication and how to set it up |
 | [03 Qualtrics How-to Guide](/03%20Qualtrics%20How-to%20Guide.md) | Learn about the Survey platform and how to access it |
 | [04 Troubleshooting](/04%20Troubleshooting.md) | General advice if you are having trouble with Okta and Qualtrics |
 | [05 Data Governance: How is Survey of Native Nations Data Managed?](/05%20Data%20Governance:%20How%20is%20Survey%20of%20Native%20Nations%20Data%20Managed%3F.md) | Learn how CICD manages and protects your data |
```
Comment (vanessa-mplsfed) [inline README.md:23]: The `05 Data Governance` link is broken

File: 05 Data Governance: How is Survey of Native Nations Data Managed?.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Handling: Only the CICD Director and relevant members of the SNN team will have full access to tribally identifiable reports and analysis. After receiving these materials, tribes (not CICD) determine whether and how to further share their information. 
+
+### Public Aggregate Analysis 
+
+Refers to: The analysis of anonymized, aggregated tribal data for research project purposes. Examples include the share of tribes collecting a particular type of tax; the average tribal expenditure on utilities; and other summary numbers. 
+
+Handling: CICD uses a cautious and conservative approach to ensure that individual tribes cannot be re-identified using released aggregate information. Analysis is typically made available to the general public through web articles and other publications.  
+
+[CICD DATA PRINCIPLES FOR RESEARCH AND DATA USE](https://www.minneapolisfed.org/indiancountry/about-us/principles-for-research-and-data-use)
```
Comment (vanessa-mplsfed) [inline 05 Data Governance: How is Survey of Native Nations Data Managed?.md:38]: Should be just CICD PRINCIPLES FOR RESEARCH AND DATA USE?

File: 05 Data Governance: How is Survey of Native Nations Data Managed?.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## DATA HANDLING: 
+Survey of Native Nations (SNN) data will be handled according to stringent internal processes at the CICD and the Federal Reserve Bank of Minneapolis. A general overview of the processes for different types of data is provided here.  
+
+### Contact Data 
+
+Refers to information used for day-to-day SNN operations. These data include survey respondents' names and email addresses, the tribe’s name, and information about the completion status of their survey.
```
Comment (vanessa-mplsfed) [inline 05 Data Governance: How is Survey of Native Nations Data Managed?.md:16]: "of ~their~ the tribe's survey."

**Review by htrostle-mplsfed — APPROVED** [review]
Yes, good to go

**Pull request merged**